app/uk/gov/hmrc/claimvatenrolmentfrontend/views/sign_up_complete_page.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.claimvatenrolmentfrontend.views.html.helpers._
+
+@this(
+        layout: templates.layout,
+        form: FormWithCSRF,
+        govukButton: GovukButton,
+        h1: h1
+)
+
+@()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@layout(Some(titleNoForm(messages("sign_up_complete.title")))) {
+
+@h1(messages("sign_up_complete.heading"))
+
+    <p class="govuk-body">@Messages("sign_up_complete.line_1")</p>
```

Comment:
I think there are paragraph helpers, could we use that here instead, e.g. @p(Html(messages("sign_up_complete.line_1")))